tests/ft_python.vader

```diff
@@ -38,3 +38,31 @@ Execute (python: pylama: errorformat):
   \ 'type': 'I',
   \ 'pattern': '',
   \ 'text': "C901 'load_library' is too complex (13) [mccabe]"}
+
+Execute (python: flake8: errorformat/postprocess: F811):
+  Save &errorformat
+  let &errorformat = neomake#makers#ft#python#flake8().errorformat
+
+  file file1.py
+  norm Iimport os
+  norm oimport os.path as os
+  lgetexpr "file1.py:2:1: F811 redefinition of unused 'os' from line 1"
+
+  AssertEqual getloclist(0), [
+  \ {'lnum': 2, 'bufnr': bufnr('%'), 'col': 1, 'valid': 1, 'vcol': 0,
+  \  'nr': 811, 'type': 'F', 'pattern': '',
+  \  'text': 'redefinition of unused ''os'' from line 1'}]
+
+  let e = copy(getloclist(0))[0]
+  call neomake#makers#ft#python#Flake8EntryProcess(e)
+  AssertEqual e, {
+  \ 'lnum': 2,
+  \ 'bufnr': bufnr('%'),
+  \ 'col': 8,
```

Comment:
The ast module (which flake8 uses) really sucks in this regard. It doesn't have allow access to the position of these sub parts of the AST. I tried to fix it there for the previous error first, but gave up because of that. I'll look into the 'os'  bug tomorrow, but it should be super rare to encounter.